### PR TITLE
Update gRPC BBEs to new project and remove generated code

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
$subject

### CODE REVIEW

**Review by BuddhiWathsala — APPROVED** [review]

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service.out [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Create a Ballerina project and a module inside it.
-# Copy generated stub file `grpc_bidirectional_streaming_pb.bal` to the module.
-# For example, if you create a module named `service`, copy the stub file to the `service` module.
+# Create a Ballerina package.
+# Copy generated stub file `grpc_bidirectional_streaming_pb.bal` to the package.
```
Comment (praneesha) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service.out:2]:
```suggestion
# Copy the generated `grpc_bidirectional_streaming_pb.bal` stub file to the package.
```

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service.out [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+# For example, if you create a package named `service`, copy the stub file to the `service` package.
 
-# Add new Ballerina file `grpc_bidirectional_streaming.bal` inside the `service` module and add service implementation.
+# Create new Ballerina file `grpc_bidirectional_streaming.bal` inside the `service` package and add service implementation.
```
Comment (praneesha) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service.out:5]:
```suggestion
# Create a new `grpc_bidirectional_streaming.bal`  Ballerina file inside the `service` package and add the service implementation.
```

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service_client.out [hunk 3]
```diff
@@ -1,11 +1,11 @@
-# Create a Ballerina project and a module inside it.
-# Copy generated stub file `grpc_bidirectional_streaming_pb.bal` to the module.
-# For example, if you create a module named `client`, copy the stub file to the `client` module.
+# Create a Ballerina package.
+# Copy generated stub file `grpc_bidirectional_streaming_pb.bal` to the package.
```
Comment (praneesha) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service_client.out:2]:
```suggestion
# Copy the generated  `grpc_bidirectional_streaming_pb.bal` stub file to the package.
```

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service_client.out [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+# For example, if you create a package named `client`, copy the stub file to the `client` package.
 
-# Add new Ballerina file `grpc_bidirectional_streaming_client.bal` inside the `client` module and add client implementation.
+# Create new Ballerina file `grpc_bidirectional_streaming_client.bal` inside the `client` package and add client implementation.
```
Comment (praneesha) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service_client.out:5]:
```suggestion
# Create a new `grpc_bidirectional_streaming_client.bal` Ballerina file inside the `client` package and add the client implementation.
```

File: examples/grpc-client-streaming/grpc_client_streaming.out [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Create new Protocol Buffers definition file `grpc_client_streaming.proto` and add service definition.
 # Run the command below in Ballerina tools distribution for stub generation.
```
Comment (praneesha) [inline examples/grpc-client-streaming/grpc_client_streaming.out:2]:
```suggestion
# Run the command below in the Ballerina tools distribution for stub generation.
```

File: examples/grpc-client-streaming/grpc_client_streaming_service.out [hunk 6]
```diff
@@ -1,11 +1,11 @@
-# Create a Ballerina project and a module inside it.
-# Copy generated stub file `grpc_client_streaming_pb.bal` to the module.
-# For example, if you create a module named `service`, copy the stub file to the `service` module.
+# Create a Ballerina package.
+# Copy generated stub file `grpc_client_streaming_pb.bal` to the package.
```
Comment (praneesha) [inline examples/grpc-client-streaming/grpc_client_streaming_service.out:2]:
```suggestion
# Copy the generated `grpc_client_streaming_pb.bal` stub file to the package.
```

File: examples/grpc-client-streaming/grpc_client_streaming_service.out [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
+# For example, if you create a package named `service`, copy the stub file to the `service` package.
 
-# Add new Ballerina file `grpc_client_streaming.bal` inside the `service` module and add service implementation.
+# Create new Ballerina file `grpc_client_streaming.bal` inside the `service` package and add service implementation.
```
Comment (praneesha) [inline examples/grpc-client-streaming/grpc_client_streaming_service.out:5]:
```suggestion
# Create a new `grpc_client_streaming.bal` Ballerina file inside the `service` package and add the service implementation.
```

File: examples/grpc-client-streaming/grpc_client_streaming_service_client.out [hunk 8]
```diff
@@ -1,11 +1,11 @@
-# Create a Ballerina project and a module inside it.
-# Copy generated stub file `grpc_client_streaming_pb.bal` to the module.
-# For example, if you create a module named `client`, copy the stub file to the `client` module.
+# Create a Ballerina package.
+# Copy generated stub file `grpc_client_streaming_pb.bal` to the package.
```
Comment (praneesha) [inline examples/grpc-client-streaming/grpc_client_streaming_service_client.out:2]:
```suggestion
# Copy the generated `grpc_client_streaming_pb.bal` stub file to the package.
```

File: examples/grpc-client-streaming/grpc_client_streaming_service_client.out [hunk 9]
```diff
@@ -1,11 +1,11 @@
...
+# For example, if you create a package named `client`, copy the stub file to the `client` package.
 
-# Add new Ballerina file `grpc_client_streaming_client.bal` inside the `client` module and add client implementation.
+# Create new Ballerina file `grpc_client_streaming_client.bal` inside the `client` package and add client implementation.
```
Comment (praneesha) [inline examples/grpc-client-streaming/grpc_client_streaming_service_client.out:5]:
```suggestion
# Create a new `grpc_client_streaming_client.bal` Ballerina file inside the `client` package and add the client implementation.
```

File: examples/grpc-secured-unary/grpc_secured_unary_service.out [hunk 10]
```diff
@@ -1,11 +1,11 @@
-# Create a Ballerina project and a module inside it.
-# Copy generated stub file `grpc_secured_unary_pb.bal` to the module.
-# For example, if you create a module named `service`, copy the stub file to the `service` module.
+# Create a Ballerina package.
+# Copy generated stub file `grpc_secured_unary_pb.bal` to the package.
```
Comment (praneesha) [inline examples/grpc-secured-unary/grpc_secured_unary_service.out:2]:
```suggestion
# Copy generated stub file `grpc_secured_unary_pb.bal` to the package.
```
Please correct all occurrences as above.

File: examples/grpc-secured-unary/grpc_secured_unary_service.out [hunk 11]
```diff
@@ -1,11 +1,11 @@
...
+# For example, if you create a package named `service`, copy the stub file to the `service` package.
 
-# Add new Ballerina file `grpc_secured_unary.bal` inside the `service` module and add service implementation.
+# Create new Ballerina file `grpc_secured_unary.bal` inside the `service` package and add service implementation.
```
Comment (praneesha) [inline examples/grpc-secured-unary/grpc_secured_unary_service.out:5]:
```suggestion
# Create new Ballerina file `grpc_secured_unary.bal` inside the `service` package and add service implementation.
```
Please correct all occurrences as above.

**Pull request merged**